webserver/script/CostBenefitAnalysis/python_code/boolean.py

```diff
@@ -0,0 +1,129 @@
+from __future__ import unicode_literals
+__author__ = 'ryan'

```

Comment:
This file is such a foreign idea in python to me.....I'm assuming we didn't write so I guess it's ok.

Reply:
Yes sorry it this isn't clear but inside the python_code directory, the only file from us (well edited file from CMU students) is UploadToPostgres. The python_code directory is a part of the cba_tool provided by Robbie. My apologies for not making that more clear.

Reply:
yes python_code directory was not written by us, only file is UploadToPostgres. Again my apologies! Thanks for going through!